pkg/arvo/app/spider.hoon

```diff
@@ -206,16 +218,28 @@
     ::
     ++  old-to-3
       |=  old=clean-slate-any
-      ^-  clean-slate
-      ?>  ?=(?(%2 %3) -.old)
-      ?:  ?=(%3 -.old)
+      ^-  clean-slate-any
+      ?>  ?=(?(%2 %3 %4) -.old)
+      ?:  ?=(?(%3 %4) -.old)
         old
```

Comment:
Might be nicer for this to produce `clean-slate-3` as per the arm name, but it might not actually matter.

Reply:
i did it like this because old-to-2 had a similar thing. and it looks like it can produce a clean-slate-4 too... i'm not sure i like the pattern but probably best to keep it consistent.